backend/src/models/game.model.ts

```diff
@@ -6,23 +6,20 @@ import { RoundSchema, Round } from "./round.model";
 
 export enum GameState {
   lobby = "LOBBY",
-  roundBefore = "ROUND_BEFORE",
-  roundPlayers = "ROUND_PLAYERS_CHOOSE",
-  roundHost = "ROUND_HOST_CHOOSE",
-  roundAfter = "ROUND_AFTER",
+  active = "ACTIVE",
```

Comment:
Ah I see why the refactor was needed lol

Reply:
Yeah lol just simplifying the state management 👌🏼